security/pass-otp/Portfile

```diff
@@ -1,24 +1,27 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tadfisher pass-otp 1.1.1 v
+github.setup        tadfisher pass-otp 1.2.0 v
+github.tarball_from releases
+
 categories          security
 platforms           darwin
 supported_archs     noarch
-license             GPL-3+
-maintainers         {loicp.eu:loic-github @lpefferkorn} openmaintainer
+license             GPL-3
+maintainers         {loicp.eu:loic-github @lpefferkorn} \
+                    {judaew.com:judaew @judaew} \
+                    openmaintainer
 
 description         A pass extension for managing one-time-password (OTP) tokens
 long_description    ${description}
 
-checksums           rmd160  850dd63a85bbf7b92a091e75601c4e37c230b73a \
-                    sha256  d5137cffa4cb9dccb38765a340be109a9ae9350979045a1f449cf72a5c4e3c1c \
-                    size    46296
+checksums           rmd160  3ac479108fa4717af3142f7468e6abb2e57622f4 \
+                    sha256  5720a649267a240a4f7ba5a6445193481070049c1d08ba38b00d20fc551c3a67 \
+                    size    46789
 
-depends_run         port:pass \
-                    port:oath-toolkit
+depends_run         port:pass port:oath-toolkit port:qrencode
```

Comment:
Please keep the style of one dependency per line. Also sort them alphabetically.